src/main/java/com/synopsys/integration/blackduck/dockerinspector/config/Config.java

```diff
@@ -184,6 +184,10 @@
     @Value("${docker.image:}")
     private String dockerImage = "";
 
+    @ValueDescription(description = "The platform (shown as 'platform' field in 'docker manifest inspect <image>' output) of the target image. You must also provide the target image (via docker.image) when using this property.  Note: when providing a platform, you may provide either the target operating system (os), the target architecture, or os/architecture.", group = Config.GROUP_PUBLIC)
+    @Value("${docker.image.platform:}")
+    private String dockerImagePlatform = "";
+
     @ValueDescription(description = "Docker tarfile path", defaultValue = "", group = Config.GROUP_PUBLIC, deprecated = false)
     @Value("${docker.tar:}")
     private String dockerTar = "";
```

Comment:
This class has a quirk inherent to its design: Our IntelliJ save actions are going to want to make dockerImagePlatform final, since there's no code setting it's value. The workaround is to set it to null down around line 708.